db/import_listener.go

```diff
@@ -15,11 +15,14 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+	"time"
 
 	sgbucket "github.com/couchbase/sg-bucket"
 	"github.com/couchbase/sync_gateway/base"
 )
 
+const MiB = 1048576
```

Comment:
This seems like it might be useful to put in `base/constants.go`? There's quite a lot of instances of `1024 * 1024` that would be clearer with this unit.